core/src/test/scala/unit/kafka/log/LogValidatorTest.scala

```diff
@@ -1094,25 +1106,6 @@ class LogValidatorTest {
       interBrokerProtocolVersion = ApiVersion.latestVersion).validatedRecords, offset)
   }
 
-  @Test(expected = classOf[InvalidRecordException])
-  def testInvalidInnerMagicVersion(): Unit = {
```

Comment:
Why get rid of this?

Reply:
It is subsumed by checkMismatchMagic above.